key_reader.py

```diff
@@ -7,16 +7,22 @@
 def read_keys():
     keys_dict = {}
     keys_file = open(KEYS_PATH, "r")
-    for line in keys_file:
-        key, value = line.strip().split(',')
-        keys_dict[key] = value
+    try:
+        for line in keys_file:
+            key, value = line.strip().split(',')
+            keys_dict[key] = value
+    except ValueError:
+        raise ValueError(f"File {KEYS_PATH} has an incorrect format.\n"
+                         "It should be a comma-delimited text, "
+                         "following the scheme:\n"
+                         "\"<key-name>,<key-value>\"")
```

Comment:
```
ValueError: File ./better_weather_keys.txt has an incorrect format.
It should be a comma-delimited text, following the scheme:
"<key-name>,<key-value>"
```
I'd say let's remove the `\n` after the `scheme` word to have it a bit more concise:
```
ValueError: File ./better_weather_keys.txt has an incorrect format.
It should be a comma-delimited text, following the scheme: "<key-name>,<key-value>"
```